docs/reference/workloads-18-19.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Workloads API changes in version 1.8 and 1.9
+approvers:
+- steveperry-53
+- kow3ns
+---
+
+## Overview
+
+The Kubernetes core workload API consists of the Deployment, DaemonSet, ReplicaSet, and StatefulSet API objects. In order to provide a stable API for users to orchestrate their workloads, we are prioritizing promoting the core workload API to GA. The batch workload API (Job and CronJob), while also important, is not part of this effort, and it will have a separate path to GA stability.
+
+- In the 1.8 release, we introduce the apps/v1beta2 API group and version. This beta version of the core workload API contains the current version of Deployment, DaemonSet,  ReplicaSet, and  StatefulSet, and it is the version we plan to promote, provided the feedback is positive, to GA in the 1.9 release.
+
+- In the 1.9 release, we plan to introduce the apps/v1 API. We intend to promote the you also mentioned that we might want to not mark everything in types.go with +DEPRECATED should we clean this up prior to approval or apps/v1beta2 group version in its entirety to apps/v1 and to deprecate apps/v1beta2 at that time.
+
```

Comment:
It looks like a comment got blended into the text here. My guess is this should say, "We intend to promote the apps/v1beta2 group version in its entirety to ..."